plugins/http_plugin/http_plugin.cpp

```diff
@@ -880,7 +764,7 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
                try {
                   my->unix_server.clear_access_channels(websocketpp::log::alevel::all);
                   my->unix_server.init_asio( &my->thread_pool->get_executor() );
-                  my->unix_server.set_max_http_body_size(my->max_body_size);
+                  my->unix_server.set_max_http_body_size(my->plugin_state->max_body_size);
```

Comment:
unix socket wasn't converted over to beast

Reply:
@spoonincode Yes this was an oversight, needs to be done

Reply:
@spoonincode Implemented, tested with keosd (uses UNIX sockets by default)